### PR TITLE
Pv2 748 add refund tests to project

### PR DESCRIPTION
**Pull request by kuzinator**

### CODE REVIEW

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner, price is changed, orginally mix funded and a negative amount is left to be paid PV2-256.feature [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+Examples: 
+    | Collection_Period         | Levy Balance |
+    | R01/Current Academic Year | 750          |
+    | R02/Current Academic Year | 375          |
+    | R03/Current Academic Year | 0            |
+
+
+
+#Levy learner, price is changed, originally mix funded and a negative amount is left to be paid - results in a refund
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner, price is changed, orginally mix funded and a negative amount is left to be paid PV2-256.feature]: aren't we removing the V1 specs when adding them?

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner, levy available, 2 providers - one provider retrospectively notifies a withdrawal need to be refunded PV2-253.feature [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    | Learner ID | Collection Period         | Delivery Period           | Levy Payments | Transaction Type |
+    | learner a  | R06/Current Academic Year | Jan/Current Academic Year | 375           | Learning         |
+
+And only the following "provider b" payments will be generated
+    | Learner ID | Collection Period         | Delivery Period           | Levy Payments | Transaction Type |
+    | learner b  | R06/Current Academic Year | Nov/Current Academic Year | -750          | Learning         |
+    | learner b  | R06/Current Academic Year | Dec/Current Academic Year | -750          | Learning         |
+    | learner b  | R06/Current Academic Year | Jan/Current Academic Year | 0             | Learning         |
+       
+#Scenario: 683-AC01- DAS learner, levy available, 2 providers - provider A and provider B, Provider B retrospectively notifies a withdrawal and previously-paid monthly instalments need to be refunded,
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner, levy available, 2 providers - one provider retrospectively notifies a withdrawal need to be refunded PV2-253.feature]: aren't we removing the V1 specs when adding them?

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner insufficient levy available to cover full payments provider retrospectively notifies of a withdrawal PV2-252.feature [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    | R06/Current Academic Year | Nov/Current Academic Year | -500          | -225                   | -25                         | Learning         |
+    | R06/Current Academic Year | Dec/Current Academic Year | -500          | -225                   | -25                         | Learning         |
+
+And only the following provider payments will be generated
+    | Collection Period         | Delivery Period           | Levy Payments | SFA Co-Funded Payments | Employer Co-Funded Payments | Transaction Type |
+    | R06/Current Academic Year | Nov/Current Academic Year | -500          | -225                   | -25                         | Learning         |
+    | R06/Current Academic Year | Dec/Current Academic Year | -500          | -225                   | -25                         | Learning         |
+
+
+#Scenario:673-AC03 DAS learner, insufficient levy available to cover full payments, provider retrospectively notifies a withdrawal and previously-paid monthly instalments need to be refunded.	Given The learner is programme only DAS
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner insufficient levy available to cover full payments provider retrospectively notifies of a withdrawal PV2-252.feature]: aren't we removing the V1 specs when adding them?

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner, price is changed, orginally mix funded and a negative amount is left to be paid PV2-256.feature (hunk 1 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner, price is changed, orginally mix funded and a negative amount is left to be paid PV2-256.feature]: we will be when business stories done, this is a tech task

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner, levy available, 2 providers - one provider retrospectively notifies a withdrawal need to be refunded PV2-253.feature (hunk 2 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner, levy available, 2 providers - one provider retrospectively notifies a withdrawal need to be refunded PV2-253.feature]: we will be when business stories done, this is a tech task

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner insufficient levy available to cover full payments provider retrospectively notifies of a withdrawal PV2-252.feature (hunk 3 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Refunds/Levy learner insufficient levy available to cover full payments provider retrospectively notifies of a withdrawal PV2-252.feature]: we will be when business stories done, this is a tech task

**Pull request merged**